build-tools/dependencies/dependencies.projitems

```diff
@@ -7,51 +7,58 @@
     </RequiredProgram>
-    <RequiredProgram Include="gettext"              Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="gettext"              Condition=" '$(HostOS)' == 'Darwin'">
       <Homebrew>gettext</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="glibtool"             Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="glibtool"             Condition=" '$(HostOS)' == 'Darwin'">
       <Homebrew>libtool</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="gsed"                 Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="gsed"                 Condition=" '$(HostOS)' == 'Darwin'">
       <Homebrew>gnu-sed</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="intltoolize"          Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="intltoolize"          Condition=" '$(HostOS)' == 'Darwin'">
       <Homebrew>intltool</Homebrew>
     </RequiredProgram>
+    <RequiredPackage Include="mingw-zlib"           Condition=" '$(HostOS)' == 'Darwin'">
+      <Homebrew>mingw-zlib</Homebrew>
+      <HomebrewTap>grendello/xawindeps</HomebrewTap>
+    </RequiredPackage>
     <RequiredProgram Include="$(JavaCPath)">
       <MinimumVersion>1.8</MinimumVersion>
       <CurrentVersionCommand                        Condition=" '$(HostOS)' != 'Windows' ">$(MSBuildThisFileDirectory)..\scripts\javac-version</CurrentVersionCommand>
       <CurrentVersionCommand                        Condition=" '$(HostOS)' == 'Windows' ">&quot;$(JavaCPath)&quot; -version 2&gt;&amp;1</CurrentVersionCommand>
       <UbuntuInstall>$(_AptGetInstall) openjdk-8-jdk</UbuntuInstall>
     </RequiredProgram>
     <RequiredProgram Include="make"                 Condition=" '$(HostOS)' != 'Windows' " />
-    <RequiredProgram Include="pkg-config"           Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="pkg-config"           Condition=" '$(HostOS)' == 'Darwin'">
       <Homebrew>pkg-config</Homebrew>
     </RequiredProgram>
-    <RequiredProgram Include="ruby"                 Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))" />
-    <RequiredProgram Include="scons"                Condition=" '$(NeedMxe)' == 'true' And $(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))">
+    <RequiredProgram Include="ruby"                 Condition=" '$(HostOS)' == 'Darwin'" />
+    <RequiredProgram Include="scons"                Condition=" '$(HostOS)' == 'Darwin'">
```

Comment:
I'm also wondering about these changes, from `scons` being *conditional* on using MXE to being *required*.

Ditto `wget`, `xz`, etc.

If I'm not building Windows binaries, I shouldn't need to install these Homebrew packages.